src/views/Onboarding.vue

```diff
@@ -199,7 +199,7 @@
                   </v-btn>
                 </div>
 
-                <div v-if="stepContentName == 'edge-installation-question'">
+                <div v-if="stepContentName === 'edge-installation-question'">
```

Comment:
@vickywane this file has become too large. Nearly 1,000 lines. It's nearly impossible to follow the logic and the interdependency between states. I spent a lot of time resolving issues with the Install PWA step, which was broken and left the use with the impression that the app was installed, when in reality it was just eating silently browser exceptions. The code in this one file became too convoluted. Please break it up into ~200 LOC files that can be independently tested.